lib/util.js

```diff
@@ -205,6 +205,22 @@ function pad(n) {
  */
 function styleText(format, text) {
   validateString(text, 'text');
+
+  if (ArrayIsArray(format)) {
+    let formatCodes = '';
+    let append = '';
+    for (const f of format) {
+      const formatCode = inspect.colors[f];
+      if (formatCode == null) {
+        validateOneOf(f, 'format', ObjectKeys(inspect.colors));
+      }
+      formatCodes += `\u001b[${formatCode[0]}m`;
```

Comment:
consider extracting `\u001b` to some const 🙂

Reply:
In my own code, I used a helper function:

```ts
function setAttrs(...attrs: Array<number>) {
  return `\x1B[` + attrs.join(';') + `m`;
}
```